plugins/woocommerce-admin/client/products/shared/simple-navigation/simple-navigation.tsx

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Icon, arrowLeft, arrowRight } from '@wordpress/icons';
+import { Link } from '@woocommerce/components';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import './simple-navigation.scss';
+
+export default function SimpleNavigation( {
```

Comment:
Maybe `PostsNavigation` or `PostNavigationLinks`?  Trying to find something more descriptive and pulling from WP terminology for next/previous links - https://codex.wordpress.org/Next_and_Previous_Links.

Reply:
done

Reply:
Ya, I'm not really certain either.  I don't love `PostsNavigation`, but was trying to find something a bit more descriptive.  I'm not super opinionated on this so if you find something else you want to change it to, that's fine with me as well.

It's not an exposed component so we can always change this easily later too.